.github/workflows/prepare_deployment_branch.yaml

```diff
@@ -0,0 +1,53 @@
+name: "Create a deployment branch in preparation of the upcoming scheduled deployment"
```

Comment:
This name may be too verbose. The workflow show up with a limited space for their name on this page: https://github.com/CDCgov/prime-reportstream/actions

```suggestion
name: "Prepare Deployment Branch"
```